contracts/staking/StakingExtension.sol

```diff
@@ -41,18 +41,16 @@ contract StakingExtension is StakingV4Storage, GraphUpgradeable, IStakingExtensi
      * initialize() function, so it uses the same access control check to ensure it is
      * being called by the Staking implementation as part of the proxy upgrade process.
      * @param _delegationUnbondingPeriod Delegation unbonding period in blocks
-     * @param _cooldownBlocks Minimum time between changes to delegation parameters, in blocks
      * @param _delegationRatio Delegation capacity multiplier (e.g. 10 means 10x the indexer stake)
      * @param _delegationTaxPercentage Percentage of delegated tokens to burn as delegation tax, expressed in parts per million
      */
     function initialize(
         uint32 _delegationUnbondingPeriod,
-        uint32 _cooldownBlocks,
+        uint32, //_cooldownBlocks, deprecated
```

Comment:
Do we need to keep it here? I understand doing it for `setDelegationParameters` so we don't change the interface but I think we can entirely remove it here?

Reply:
We could, but I'd rather keep it to keep changes minimal (this is called from Staking.initialize, so we'd have to change that too)